book/_toc.yml

```diff
@@ -10,4 +10,5 @@ parts:
   chapters:
   - file: models/mlp
   - file: models/hierarchical_bnn.md
-  - file: models/LogisticRegression.md
\ No newline at end of file
+  - file: models/LogisticRegression.md
+  - file: models/sparse_regression
```

Comment:
should they all have `.md` extension explicitly spell out?

Reply:
It doesn't matter, but having a uniform naming style doesn't hurt; I'll make the change here.